oterr/constants.go

```diff
@@ -0,0 +1,32 @@
+// Copyright 2019, OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package oterr
```

Comment:
I would avoid using project name or abbreviation of project name in packages. It creates a lot of source code renaming work if the project gets renamed in the future (e.g. we had to do it when we moved from OpenCensus from OpenTelemetry).
I would name this package simply `errors` or similar.

Reply:
Looks like we had a similar discussion about this previously: https://github.com/open-telemetry/opentelemetry-service/pull/191#issuecomment-514432106.

I don't have a preference here - WDYT? /cc @pjanotti @owais

Reply:
OK, I did not realize this is already an existing package. I am OK with moving forward and merging as is since it looks like this was discussed and accepted as the best we can do.

Reply:
Sounds good, I opened an issue https://github.com/open-telemetry/opentelemetry-service/issues/247 to keep track. If we got a better suggestion here we can send a separate PR to rename this package.